libraries/joomla/database/driver.php

```diff
@@ -162,20 +145,6 @@ public static function isSupported();
 	protected $utf8mb4 = false;
 
 	/**
-	 * @var         integer  The database error number
-	 * @since       11.1
-	 * @deprecated  12.1
-	 */
-	protected $errorNum = 0;
-
-	/**
-	 * @var         string  The database error message
-	 * @since       11.1
-	 * @deprecated  12.1
-	 */
-	protected $errorMsg;
```

Comment:
These two class vars are still used throughout the drivers, especially in the execute methods.